terraform/customer_simple_registration.tf

```diff
@@ -91,6 +91,7 @@ resource "docker_image" "registration" {
 resource "docker_container" "registration" {
   name  = "simple-registration"
   image = docker_image.registration.image_id
+  network_mode = "bridge"
```

Comment:
do you know why I didn't need this? do i have something configured differently in my local docker settings?

Reply:
not 100% sure, I just know it triggered a re-spin of the container every time for me without that flag.